src/dac.rs

```diff
@@ -0,0 +1,21 @@
+//! Digital-analog conversion traits
+
+
+//! A trait used to identify a digital-to-analog converter, and its
+//! most fundamental features.
+
+pub trait DacTrait<E> {
```

Comment:
As multiple channels are not modelled, I think it would make sense to rather name this something like `SingleChannelDac`.